tests/superphot_plus/test_ztf_transient_fit.py

```diff
@@ -20,3 +25,31 @@ def test_dynesty_single_file(tmp_path, single_ztf_lightcurve_compressed):
     params = np.load(output_file)["arr_0"]
     ## could be between ~600 and ~800, and can vary based on hardware.
     assert 600 <= len(params) <= 800
+
+    # Check that the same means are reasonably close (within 10% relative value).
+    # Despite setting the the random seed, we still need to account for floating
+    # point differences, etc.
+    expected = [
+        1035.0,
+        0.005,
+        13.5,
+        -4.8,
+        4.0,
+        23.4,
+        0.03,
+        1.1,
+        1.0,
+        1.0,
+        1.0,
+        0.96,
+        0.56,
+        0.87,
+    ]
+    assert len(expected) == len(sample_mean)
+    for i in range(len(expected)):
```

Comment:
So this is checking that all of the outputs are within 10% of the expected value? Does something like this also work?

```
assert np.isclose(sample_mean, expected, rtol=.1)
```

Reply:
Yes. That works better (and will be correctly vectorized).